### PR TITLE
Mint page

### PR DESCRIPTION
**Pull request by towardsnewtech**

### CODE REVIEW

**Review by SweetmanTech — APPROVED** [review]
ship it 🚢

File: components/Footer/Footer.tsx [hunk 1]
```diff
@@ -69,7 +69,7 @@ const Footer: React.FC<FooterProps> = ({ className }) => {
         className || ""
       } footer`}
     >
-      <div className="md:w-[1280px] w-[90%]">
+      <div className="w-[90%] xl:w-[1280px] xl:!max-w-[1280px]">
```
Comment (sweetmantech) [inline components/Footer/Footer.tsx]: looks amazing

Do you need the `:!max-w`? defining both breakpoints should work without. right?

as is, looks awesome

**Review by SweetmanTech — APPROVED** [review]
ship it 🚢

File: components/DesktopMenu/DesktopMenu.tsx [hunk 2]
```diff
@@ -17,32 +21,45 @@ const DesktopMenu = () => {
...
   }
 
+  useEffect(() => {
+    if (router.pathname.includes("/mint")) {
+      setIsHidden(true)
+      return
+    }
+
+    setIsHidden(false)
+  }, [router.pathname])
```
Comment (sweetmantech) [inline components/DesktopMenu/DesktopMenu.tsx]: do we need a `useEffect` here?

Comment (sweetmantech) [reply components/DesktopMenu/DesktopMenu.tsx]: what happens if you just define, after `line 15`

`const isHidden = router.pathname.includes("/mint")?`

Comment (sweetmantech) [reply components/DesktopMenu/DesktopMenu.tsx]: and remove both the `useState` and `useEffect` hooks?

File: components/MenuList/MenuList.tsx [hunk 3]
```diff
@@ -20,6 +24,15 @@ const MenuList = ({ toggleMenu }) => {
...
   }, [themeMode])
 
+  useEffect(() => {
+    if (router.pathname.includes("/mint")) {
+      setIsHidden(true)
+      return
+    }
+
+    setIsHidden(false)
+  }, [router.pathname])
```
Comment (sweetmantech) [inline components/MenuList/MenuList.tsx]: same here as in `DesktopMenu`

remove `useEffect` & `useState`

define `isHidden` after line 14.

File: components/DesktopMenu/DesktopMenu.tsx [hunk 4]
```diff
@@ -19,30 +24,34 @@ const DesktopMenu = () => {
...
+      {!isHidden && (
+        <div className="flex items-center pr-4">
+          <Link href="/status" target="_blank" rel="noreferrer">
+            <div className="font-bold cursor-pointer dark:text-white text-black">
+              Allowlist Status
+            </div>
+          </Link>
+        </div>
+      )}
+      {!isHidden && (
```
Comment (crazzywizard) [inline components/DesktopMenu/DesktopMenu.tsx]: Is there a reason this can't be moved to the above conditional block?

Comment (towardsnewtech) [reply components/DesktopMenu/DesktopMenu.tsx:36]: will update soon.

File: components/DesktopMenu/DesktopMenu.tsx [hunk 5]
```diff
@@ -10,6 +11,10 @@ import { useTheme } from "../../providers/ThemeProvider"
 const DesktopMenu = () => {
   const { onChangeThemeConfig, themeMode } = useTheme()
 
+  const router = useRouter()
+
+  const isHidden = router.pathname.includes("/mint")
```
Comment (sweetmantech) [inline components/DesktopMenu/DesktopMenu.tsx:16]: 💯

**Review by SweetmanTech — APPROVED** [review]
ship it 🚢

**Review by SweetmanTech — APPROVED** [review]
ship it 🚢

**Review by SweetmanTech — APPROVED** [review]
ship it 🚢

**Review by SweetmanTech — APPROVED** [review]
ship it 🚢

**Pull request merged**